storage/allocator.go

```diff
@@ -251,9 +251,9 @@ func (a *Allocator) AllocateTarget(required roachpb.Attributes, existing []roach
 			return target, nil
 		}
 		if len(attrs) == 0 {
-			return nil, util.Errorf("unable to allocate a target store; no candidates available")
+			return nil, util.Errorf("no suitable replication target store found, are you running enough nodes?")
```

Comment:
It would be good to fix this error messages (and others) that occur when a cluster is running with fewer than the required number of nodes. These error messages can be considered part of the UI and right now they are giving a poor experience.